C-Plus-Plus/cp/Infix_to_postfix.md

```diff
@@ -0,0 +1,19 @@
+# Conversion of an infix expression to postfix expression.
```

Comment:
No need of this file. I have made appropriate changes to the main README.md file to link your code.

Reply:
@atarax665  i removed the Infix_to_postfix.md.